packages/transport/src/modules/challenges/ChallengeController.ts

```diff
@@ -31,9 +31,9 @@ export class ChallengeController extends TransportController {
     private async validateChallengeLocally(challenge: Challenge, signedChallenge: ChallengeSigned): Promise<{ isValid: boolean; correspondingRelationship?: Relationship }> {
         if (!challenge.createdBy) return { isValid: false };
 
-        const relationship = await this.parent.relationships.getActiveRelationshipToIdentity(challenge.createdBy);
+        const relationship = await this.parent.relationships.getOnceActiveRelationshipToIdentity(challenge.createdBy);
         if (!relationship) {
-            throw TransportCoreErrors.general.recordNotFound(Relationship, challenge.createdBy.toString());
+            throw TransportCoreErrors.general.recordNotFound("Once active Relationship to Identity", challenge.createdBy.toString());
```

Comment:
I think you forgot to undo this

Reply:
I'm not sure if we want to prevent validating challenges after relationship termination - usually challenges are very short-lived but idk

Reply:
maybe we should also discuss this to @stnmtz on monday.

Reply:
as discussed you should remove this.